src/utils/update/trusts.ts

```diff
@@ -0,0 +1,12 @@
+import { ApplicationData } from "model";
+
+export const checkHasAnyBosWithTrusteeNocs = (appData: ApplicationData) => {
+  const bosThatCanHaveTrusteeNoc = [
+    ...(appData.beneficial_owners_individual || []),
+    ...(appData.beneficial_owners_corporate || []),
+    ...(appData.update?.review_beneficial_owners_individual || []),
```

Comment:
we should not be checking the review ones. They are temporary and moved into main list once reviewed!

Reply:
Inclined to leave as is - does no harm in checking both.

And in the no change journey, where this is currently called, the beneficial owners are actually still in the review lists.

Reply:
ok, it's important to know. We should really have separate checks for each set of lists. review and reviewed

Reply:
Maybe just go over review lists then?

Reply:
> ok, it's important to know. We should really have separate checks for each set of lists. review and reviewed

Should we? Currently no need for checking them independently here. 

> Maybe just go over review lists then?

As above, I'm inclined to leave it as is. Right now we just need to know if there are _any_ BOs with trustee NOC; where they are doesn't matter.